ChopChop/Storage/Models/Ingredients/OnlineIngredientRecord.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  OnlineIngredientRecord.swift
+//  ChopChop
+//
+//  Created by Cao Wenjie on 27/3/21.
+//
+import FirebaseFirestore
+
+struct OnlineIngredientRecord {
+    var name: String
+    var quantity: QuantityRecord
+}
+
+extension OnlineIngredientRecord: Codable {
+}
+
+extension OnlineIngredientRecord {
+    func toRecipeIngredient() throws -> RecipeIngredient {
```

Comment:
tbh i think it should be the inverse, ie Models should be intiialise with record
an e.g. would be the `required init(row: Row)` method. this initiliases a runtime model with a row in the databae